cocos/core/animation/marionette/animation-controller.ts

```diff
@@ -37,7 +37,7 @@ export class AnimationController extends Component {
 
     private _graphEval: AnimationGraphEval | null = null;
 
-    public start () {
+    public onLoad () {
```

Comment:
What's problem if it's in start phase

Reply:
Many applications invoke animation controller API at their start method.

Reply:
User maybe invoke these API  in `onLoad` method, so `__preload` seems better